### PR TITLE
Supply CSAR examples

### PR DESCRIPTION
**Pull request by anzoman**
No more huge PRs :smiley:.With this PR the extracted TOSCA 
CSAR examples will be introduced as a separate folder.

### CODE REVIEW

File: cloud/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Cloud xOpera examples
+This folder contains xOpera examples that can be used to deploy services on different popular cloud providers. We try to
+provide simple blueprints that show the creation and usage of different cloud resources with xOpera and Ansible. Since
+these examples are in general more complex, they cannot be deployed directly since they require some prerequisites to be
```
Comment (cankarm) [inline cloud/README.md:4]: _... complex, they require more attention to satisfy the prerequisites._

File: cloud/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+satisfied first. This means that you will need your cloud account to deploy the solution to a specific cloud provider
+and also you may need to install some other accompanying tools that are used for the deployment process.
```
Comment (cankarm) [inline cloud/README.md:6]: Could this be solved with TOSCA? So you have the "platform node" as for example is done in particles?

File: csars/small/inputs.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+    "host_ip": "localhost",
+    "relationship_input": "Hey, I am in relationship!"
```
Comment (cankarm) [inline csars/small/inputs.json:3]: .. in **a** relationship? :D

**Review by cankarm — COMMENTED** [review]
Thanks, Anže, please check some comments.

File: cloud/README.md (hunk 2 above)
Comment (anzoman) [reply cloud/README.md:6]: Sure, this be simplified like in particles. I propose doing this in one of the following PRs.

File: csars/small/inputs.json (hunk 3 above)
Comment (anzoman) [reply csars/small/inputs.json]: Resolved :D

File: cloud/README.md (hunk 1 above)
Comment (anzoman) [reply cloud/README.md]: Refactored.

**Review by cankarm — APPROVED** [review]
It seems OK now.

**Pull request merged**